atoms/colors/index.css

```diff
@@ -1,3 +1,4 @@
+@import "/foundation/all.css";
```

Comment:
can remove these imports for now I need to figure out better way this is creating problem. Need to look into it.